doc/get-started.md

```diff
@@ -0,0 +1,102 @@
+# Getting started
+
+## Setting up the environment
+
+1. Make sure your system and Tizen device meet the minimum requirements in [README](../README.md#system-requirements) and all the required tools are installed.
```

Comment:
Sequential numbering not explicitly increasing in list

Reply:
This is legal.
https://guides.github.com/features/mastering-markdown/